docs/usage/configuration-options.md

```diff
@@ -689,6 +689,10 @@ This is caused by a race condition in GitLab's Merge Request API - [read the cor
 
 Configuration added here applies for all Go-related updates, however currently the only supported package manager for Go is the native Go Modules (the `gomod` manager).
 
+For self-hosted users, `GOPROXY`, `GONOPROXY` and `GOPRIVATE` environment variables are supported.
+
+But when you use the `direct` or `off` keywords Renovate will fallback to its own fetching strategy (i.e. directly from GitHub, etc).
```

Comment:
Is this `direct` and `off` keywords related to something from `gomod` manager, or is this Renovate specific.
To me it's not clear what things `direct` and `off` affect, and how/where you use them.

But maybe this is clear right away for somebody who uses `gomod`?

Reply:
Yes, that are go proxy specific values. So anybody who is using these env values should know about them. 😉

Reply:
I think it should be clear to every one, because not admins need to understand these even if they are not go devs themselves. So let's go overboard on documentation for this

Reply:
@viceice That link you've supplied does not directly mention the `direct` and/or `off` keywords though.